.github/workflows/pipeline.yml

```
@@ -0,0 +1,49 @@
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'npm'
+        cache-dependency-path: backend/package-lock.json
+        
+    - name: Install dependencies
+      working-directory: ./backend
+      run: npm ci
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)